site/docs/components/drawer/accessibility.mdx

```diff
@@ -7,33 +7,29 @@ data:
   $ref: ./#/data
 ---
 
-### Best practices
-
-You can customize or disable the animations using the `prefers-reduced-motion` CSS media feature.
-
 ### Keyboard interactions
 
 <KeyboardControls>
 <KeyboardControl keyOrCombos="Enter / Space">
 
-- This action activates the trigger/disclosure button if it is in focus, controlling the visibility of the drawer.
-- In closed state: The trigger is a user-defined content element. Drawer opens and focus is set to first focusable element in the Drawer.
-- In open state: The trigger is the Drawer's 'X' button. Drawer closes and focus returns to content element trigger.
+- If focus is on the trigger element, this action activates the element and opens the drawer. Focus doesn't leave the drawer.
+- If focus is on close button, this action closes the drawer. Focus returns to the trigger element.
+- If focus is on a content element configured to close the drawer when activated, such as a _Submit_ button, this action closes the drawer.
 
 </KeyboardControl>
 <KeyboardControl keyOrCombos="Tab">
 
-- Tab moves focus on to the next focusable element in the drawer. When focus is on the last element in the drawer, it moves focus to first focusable element in the drawer.
+- Tab moves focus through focusable content elements in the drawer. Focus doesn't leave the drawer.
 
 </KeyboardControl>
 <KeyboardControl keyOrCombos="Shift + Tab">
 
-- This action moves focus to the previous focusable element in the drawer. When focus is on the first element in a drawer, it moves focus to last focusable element in the drawer.
+- This action moves focus to last focusable content element in the drawer. Focus doesn't leave the drawer.
```

Comment:
This isn't true. Shift + Tab moves focus backwards through the webpage. Reading this it sounds like Shift + Tab jumps you to the last element.

Reply:
good shout - updated to say previous instead of last